# Request
string  data
bool    publish
---
# Response
int32  bytesSent
